src/views/quote_view.js

```diff
@@ -0,0 +1,34 @@
+import Quote from "../models/quote";
+import Backbone from 'backbone';
+
+const QuoteView = Backbone.View.extend({
+  initialize(params) {
+    this.template = params.template;
+    this.bus = params.bus
+    this.listenTo(this.model, "change", this.render);
+  },
+
+  render() {
+    const compiledTemplate = this.template(this.model.toJSON());
+    this.$el.html(compiledTemplate);
+    return this;
+  },
+
+  events: {
+    'click button.btn-buy': 'buyQuote',
+  },
+
+
+  buyQuote(event) {
+    this.model.buy();
+    this.bus.trigger('bought_quote', this.model);
```

Comment:
I think in this case we should trigger the event before calling `buy()`. This is not intuitive: we shouldn't trigger the event before the thing has actually happened, right?

However, in this case it's important because `buy()` will change the price of the quote, and we want the trade to be recorded with the original price. Right now if the price is $90.00 and I hit buy, the trade is recorded showing that I bought the quote at $91.00.